lib/tran_pipe.c

```diff
@@ -73,6 +73,10 @@ tran_pipe_send_iovec(int fd, uint16_t msg_id, bool is_reply,
 
     ret = writev(fd, iovecs, nr_iovecs);
 
+    /* Quieten static analysis. */
```

Comment:
Curious, what's this about?

Reply:
iovec[0] refers to stack, and clang thinks we might try to use it.